src/file-upload/__tests__/file-upload.test.tsx

```diff
@@ -181,9 +181,20 @@ describe('FileUpload input', () => {
 
   test('file input fires onChange with files in details', () => {
     const wrapper = render({ multiple: true });
-    fireEvent(wrapper.findNativeInput().getElement(), new CustomEvent('change', { bubbles: true }));
+    const input = wrapper.findNativeInput().getElement();
+    Object.defineProperty(input, 'files', { value: [file1, file2] });
+    fireEvent(input, new CustomEvent('change', { bubbles: true }));
 
-    expect(onChange).toHaveBeenCalledWith(expect.objectContaining({ detail: { value: [] } }));
+    expect(onChange).toHaveBeenCalledWith(expect.objectContaining({ detail: { value: [file1, file1] } }));
```

Comment:
We're selecting two different files (file1 and file2), but the `onChange` event here provides an array with only `file1` (but twice). I'm a bit confused by this, I would have expected that `onChange` returns _both_ files. Is this intentional and if so can we document it with an inline comment?

Reply:
good catch, it should indeed be `[file1, file2]`... updated the test to _actually_ test what I thought it was testing